hail/python/hail/methods/impex.py

```diff
@@ -587,6 +596,14 @@ def import_locus_intervals(path, reference_genome='default', skip_invalid_interv
         Interval-keyed table.
     """
 
+    if contig_recoding is not None:
+        contig_recoding = hl.literal(contig_recoding)
+
+    def recode_contig(x):
+        if contig_recoding is None:
+            return x
+        return contig_recoding[x]
```

Comment:
This should be the equivalent of `.get`. The dictionary of recordings doesn't have to have all contigs.

Reply:
Good catch, I added tests of both cases where the file has a mix of `chr22` and `22`.